specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2022-12-01-preview/examples/PrivateEndpointConnections_Create_MaximumSet_Gen.json

```diff
@@ -0,0 +1,80 @@
+{
+  "title": "PrivateEndpointConnections_Create",
+  "operationId": "PrivateEndpointConnections_Create",
+  "parameters": {
+    "subscriptionId": "55980C3D-EC3D-44F3-97C2-1E47DBA30921",
+    "resourceGroupName": "rgelasticsan",
+    "elasticSanName": "aev2h_9H-xUDkxV",
+    "privateEndpointConnectionName": "briclbdjatewdyhkzorchezphzz",
+    "parameters": {
+      "properties": {
+        "privateEndpoint": {},
+        "privateLinkServiceConnectionState": {
+          "status": "Pending",
+          "description": "pi",
+          "actionsRequired": "aotylexvkyxwpzwioisvvfcb"
+        },
+        "groupIds": [
+          "55980C3D-EC3D-44F3-97C2-1E47DBA30921"
+        ]
+      },
+      "tags": {
+        "key3005": "aicncexsckwnjlcnpru"
+      }
+    },
+    "api-version": "2022-12-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "provisioningState": "Invalid",
+          "privateEndpoint": {
+            "id": "cyyupaqfbsibqgtpnwsqrwnjop"
+          },
+          "privateLinkServiceConnectionState": {
+            "status": "Pending",
+            "description": "pi",
+            "actionsRequired": "aotylexvkyxwpzwioisvvfcb"
+          },
+          "groupIds": [
+            "55980C3D-EC3D-44F3-97C2-1E47DBA30921"
+          ]
+        },
+        "id": "xxnomshlufmq",
+        "name": "bvijkqvtpj",
+        "type": "ndc",
+        "tags": {
+          "key3005": "aicncexsckwnjlcnpru"
+        }
+      }
+    },
+    "201": {
+      "headers": {
+        "Azure-AsyncOperation": "https://contoso.com/operationstatus"
+      },
+      "body": {
+        "properties": {
+          "provisioningState": "Invalid",
+          "privateEndpoint": {
+            "id": "cyyupaqfbsibqgtpnwsqrwnjop"
```

Comment:
The ID still doen't have correct format.
There's a sample https://github.com/Azure/azure-rest-api-specs/blob/02acc29a5618a6998469a7fcc750e301f130a19f/specification/storage/resource-manager/Microsoft.Storage/stable/2022-09-01/examples/StorageAccountPutPrivateEndpointConnection.json